tests/test_pricing_model.py

```diff
@@ -11,8 +11,8 @@
 import unittest
```

Comment:
Might want to consider making this several files so that it is easier to find stuff